jobs/build-kola-containers.Jenkinsfile

```diff
@@ -186,12 +186,16 @@ lock(resource: "build-kola-containers") {
                         images += " --image=docker://${params.CONTAINER_REGISTRY_STAGING_REPO}:${imageName}-${arch}-${shortcommit}"
                     }
 
-                    shwrap("""
-                        export STORAGE_DRIVER=vfs # https://github.com/coreos/fedora-coreos-pipeline/issues/723#issuecomment-1297668507
```

Comment:
if somehow something changed and we were able to go back to pushing in an openshift pod would we still need `STORAGE_DRIVER=vfs` ?

Reply:
I know you asked about this over in https://github.com/cri-o/cri-o/pull/8514#issuecomment-2417534158 but I wonder if there is a more official place to ask and get an answer.